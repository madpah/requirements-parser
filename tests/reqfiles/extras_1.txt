MyPackage[PDF]==3.0
Fizzy [foo, bar]
