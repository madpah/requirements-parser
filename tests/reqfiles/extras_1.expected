[
 {
  "specifier": true, 
  "local_file": false, 
  "name": "MyPackage", 
  "editable": false, 
  "uri": null, 
  "extras": [
   "pdf"
  ], 
  "vcs": null, 
  "path": null, 
  "line": "MyPackage[PDF]==3.0", 
  "specs": [
   [
    "==", 
    "3.0"
   ]
  ], 
  "revision": null
 }, 
 {
  "specifier": true, 
  "local_file": false, 
  "name": "Fizzy", 
  "editable": false, 
  "uri": null, 
  "extras": [
   "foo", 
   "bar"
  ], 
  "vcs": null, 
  "path": null, 
  "line": "Fizzy [foo, bar]", 
  "specs": [], 
  "revision": null
 }
]